lib/phobos.rb

```diff
@@ -77,16 +78,20 @@ def configure_logger
         appenders << Logging.appenders.file(log_file, layout: json_layout)
       end
 
-      @ruby_kafka_logger = nil
+      @ruby_kafka_logger = config.custom_kafka_logger
 
-      if ruby_kafka
```

Comment:
Do you think it would be a good idea to describe (and protect) your changes with some specs?

Reply:
Yep, I can do that. Wanted to put up the PR for an initial look first.